src/coreclr/jit/codegenlinear.cpp

```diff
@@ -1807,7 +1807,6 @@ void CodeGen::genConsumePutStructArgStk(GenTreePutArgStk* putArgNode,
 #if FEATURE_ARG_SPLIT
 //------------------------------------------------------------------------
 // genConsumeArgRegSplit: Consume register(s) in Call node to set split struct argument.
-//                        Liveness update for the PutArgSplit node is not needed
```

Comment:
Can you update the method name in the comment?

Reply:
Sure, but let me do it as part of another change to avoid rerunning CI.